config/default.yml

```diff
@@ -589,6 +589,12 @@ PreCommit:
     include:
       - '**/*.html'
 
+  LineEndings:
+    description: 'Check line endings'
+    enabled: false
+    eol: 'lf'
+    # eol: 'crlf'
```

Comment:
It looks like you changed from `unix` / `windows` to `lf` / `crlf` in a subsequent commit. Any reason we shouldn't go with just listing the literal characters, i.e. `\n` and `\r\n`?

Looking at the list of possible [line ending representations](https://en.wikipedia.org/wiki/Newline#Representations), I'm skeptical we'll _ever_ have to support anything besides what are now called Unix and Windows line endings, but if we allow specifying the literal characters in the option then we're open to supporting any line ending.